tests/Test/Pager/Pagination/ForceValidParametersOptionTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Test\Pager\Pagination;
+
+use Knp\Component\Pager\Paginator;
+use Test\Tool\BaseTestCase;
+
+final class ForceValidParametersOptionTest extends BaseTestCase
+{
+    /**
+     * @test
+     */
+    public function shouldBeAbleToReplaceInvalidLimit(): void
+    {
+        $p = new Paginator;
+
+        $items = \range(1, 23);
+        $view = $p->paginate($items, 1, 0, ['forceValidPaginationParameters' => true]);
+
+        $view->renderer = function ($data) {
+            return 'custom';
+        };
+        $this->assertEquals('custom', (string)$view);
```

Comment:
Do you really need this block? It looks like it tests functionality that is entirely orthogonal